sphinx/templates/quickstart/conf.py_t

```diff
@@ -101,24 +111,6 @@ html_static_path = ['{{ dot }}static']
 # 'searchbox.html']``.
 #
 # html_sidebars = {}
-
-
-# -- Options for Epub output -------------------------------------------------
-
-# Bibliographic Dublin Core info.
-epub_title = project
-
-# The unique identifier of the text. This can be a ISBN number
-# or the project homepage.
-#
-# epub_identifier = ''
-
-# A unique identification for the text.
-#
-# epub_uid = ''
-
-# A list of files that should not be packed into the epub file.
-epub_exclude_files = ['search.html']
```

Comment:
At present, epub3 builder has no longer generated `search.html`. So this setting is meaningless.